tutorials/hoon/arms-and-cores.md

```diff
@@ -410,7 +410,7 @@ This almost looks like a function call of sorts.
 
 Let's take a quick look at the battery of one core in the dojo to show that this is true by inputting one into the dojo.
 
-```hoon
+```
```

Comment:
Why did you remove the hoon tag here? does it color dojo stuff weirdly?

Reply:
As near as I've been able to tell, 95%+ of the time dojo output appear in the docs it is without the hoon tag. So I've been removing the other 5% when I see them. They are technically different things. I think if we want to also highlight dojo output we should create a separate color scheme for it to emphasize the difference.

Reply:
In general when I was seeding the code blocks with metadata I skipped all the Dojo prompts; Dojo syntax isn't identical to Hoon syntax and I wanted to avoid potentially wonky results. I don't know. I tried this code block in my VS Code's Hoon syntax highlighting and it looked alright. So maybe I was off.